dart/lib/src/sentry.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:meta/meta.dart';
+
+import 'client.dart';
+import 'protocol.dart';
+import 'sentry_options.dart';
+import 'stack_trace.dart';
+
+/// Sentry SDK main entry point
+///
+class Sentry {
+  static SentryClient _client;
+
+  Sentry._();
+
+  static void init(SentryOptions options) {
+    _client = SentryClient(
+      dsn: options.dsn,
+      environmentAttributes: options.environmentAttributes,
+      compressPayload: options.compressPayload,
+      httpClient: options.httpClient,
+      clock: options.clock,
+      uuidGenerator: options.uuidGenerator,
+    );
+  }
+
+  /// Initializes the SDK
+  static void initDns(String dns) => _client = SentryClient(dsn: dns);
+
+  /// Reports an [event] to Sentry.io.
+  static Future<SentryResponse> captureEvent(
+    Event event, {
+    StackFrameFilter stackFrameFilter,
+  }) async {
+    return _client.captureEvent(
+      event: event,
+      stackFrameFilter: stackFrameFilter,
+    );
+  }
```

Comment:
our APIs return either a https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/protocol/SentryId.java
or an empty `SentryId` which is `00000000-0000-0000-0000-000000000000` I guess.
Also we never take a `stackFrameFilter` as a param, we can work on this stuff later, but this will need to be changed.

Reply:
should I rename the `SentryResponse` class https://github.com/getsentry/sentry-dart/blob/f63ac278c1de1251a466920e0bd7d5c082477e34/dart/lib/src/client.dart#L234-L256 to `SentryId`?

Reply:
no, not really, this is the whole response, we'd need to create a `SentryId` class and inside of the client, we just read the `uuid` out of the response and create a `SentryId` with the proper `uuid`, this doesn't need to be in this PR